lib/license_finder/license/text.rb

```diff
@@ -6,15 +6,28 @@ module Text
       SPACES = /\s+/.freeze
       QUOTES = /['`"]{1,2}/.freeze
       PLACEHOLDERS = /<[^<>]+>/.freeze
+      SPECIAL_SINGLE_QUOTES = /[‘’]/.freeze
+      SPECIAL_DOUBLE_QUOTES = /[“”„«»]/.freeze
+      ALPHABET_ORDERED_LIST = /\\\([a-z]\\\)\\\s/.freeze
+      ALPHABET_ORDERED_LIST_OPTIONAL = '(\([a-z]\)\s)?'
+      LIST_BULLETS = /(\d{1,2}\\\.|\\\*)\\\s/.freeze
+      LIST_BULLETS_OPTIONAL = '(\d{1,2}.|\*)\s*'
 
       def self.normalize_punctuation(text)
-        text.gsub(SPACES, ' ')
-            .gsub(QUOTES, '"')
-            .strip
+        text.dup.force_encoding('UTF-8')
+                 .gsub(SPECIAL_DOUBLE_QUOTES, '"')
```

Comment:
I also found some licenses using quote characters or comment characters and also escape quotes already.
What do you think additional about
```suggestion
                 .gsub(/\>+/, '')
                 .gsub(/\*+/, '')
                 .gsub(/\\\"/, '"')
                 .gsub(SPECIAL_DOUBLE_QUOTES, '"')
```